### DISCUSSION TITLE
How to custom font family of pdf reader? I want default is sans

### DISCUSSION THREAD

**Discussion by TYZRPVX** [RyotaUshio/obsidian-pdf-plus#289] [Q&A] [answered]
in pdf plus, it's serif
 ![image](https://github.com/user-attachments/assets/a928ef0a-9fc8-42c2-8fa7-ef36f75293fb)

in MS edge, it's sans
![image](https://github.com/user-attachments/assets/7ea526c5-5b63-479c-9486-ab1fdd8ec620)

**Comment by RyotaUshio** [comment, marked as answer]
As I explained in the documentation and readme, PDF++ does not provide an original PDF reader; it just uses Obsidian's builtin reader. Does this also happen when PDF++ is disabled? If so, you should open a bug report or a feature request in the Obsidian forum. Attaching the PDF file will help them identify the problem. Let me know if it's actually PDF++ specific.
